### PR TITLE
Staggered text components

### PR DESCRIPTION
**Pull request by Ajknight121**
Resolves issues #138
There are now two staggered text components. 
StaggeredTextDivs will take entire react divs as a prop and will add a text alignment style.
StaggeredTextObjs will take simplified object with header and body text and apply text alignment styling. StageTextObj can be expanded further for easier section creation by adding more optional object properties.

### CODE REVIEW

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
Could you separate out the examples outside the main file? We have a couple examples of this where we just use a js file with an object in it.

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
Two more things, could you change the default header to h2, and when you create the extra js file, just add the example as a comment and leave an empty object. Also remove it from any pages ATM. Thanks!

File: src/components/StaggeredSections/staggeredSections.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        paddingLeft: 15,
+        paddingBottom:10,
+        textAlign: "left",
+    }
+        const rightSide = {
+        marginRight: 0,
+        paddingRight: 15,
+        paddingBottom: 10,
+        textAlign: "right",
+    }
```
Comment (ChristianBingman) [inline src/components/StaggeredSections/staggeredSections.js:36]: Could you switch these to tailwind instead.

File: src/components/StaggeredSections/staggeredSections.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+        const sectionItems = []
+        props.sections.forEach((section, i) => {
+        sectionItems.push(
+        <section key = {section.id + "1"} style = {(i % 2 === 0) ? leftSide : rightSide}>
```
Comment (ChristianBingman) [inline src/components/StaggeredSections/staggeredSections.js:41]: Could you add a bit more spacing between the items?

File: src/pages/about.js [hunk 3]
```diff
@@ -4,9 +4,13 @@ import ExecutiveLeaders from '../components/ExecutiveLeaders';
 import Description from '../components/Description';
 import ArchiveCarousel from '../components/ArchiveCarousel';
 import WhoAreWe from '../components/WhoAreWe';
-
+import StaggeredSections from "../components/StaggeredSections/staggeredSections";
+import {exampleObjects} from "../api/exampleStaggeredText/exampleOBJ";
+import {exampleDivs} from "../api/exampleStaggeredText/exampleDIV";
```
Comment (ChristianBingman) [inline src/pages/about.js:9]: Even if they are examples, they do need to be in a specific place, so the term example should probably not be prepended here.

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
Just a couple changes.

File: src/components/StaggeredSections/staggeredSections.js (hunk 2 above)
Comment (Ajknight121) [reply src/components/StaggeredSections/staggeredSections.js:41]: Space the code or space the content?

File: src/components/StaggeredSections/staggeredSections.js (hunk 1 above)
Comment (Ajknight121) [reply src/components/StaggeredSections/staggeredSections.js:36]: className expects a CSSProperties and it fails when assigning a tailwind string

File: src/components/StaggeredSections/staggeredSections.js (hunk 2 above)
Comment (ChristianBingman) [reply src/components/StaggeredSections/staggeredSections.js:41]: Space between each section

File: src/components/StaggeredSections/staggeredSections.js (hunk 1 above)
Comment (ChristianBingman) [reply src/components/StaggeredSections/staggeredSections.js:36]: Ok, then no worries

File: src/components/StaggeredSections/staggeredSections.js (hunk 2 above)
Comment (Ajknight121) [reply src/components/StaggeredSections/staggeredSections.js:41]: The margin has been increased

**Review by ChristianBingman — APPROVED** [review]
LGTM!

**Pull request merged**